parts/configure-swarm-cluster.sh

```diff
@@ -167,7 +168,9 @@ retrycmd_if_failure() { for i in 1 2 3 4 5; do $@; [ $? -eq 0  ] && break || sle
 installDocker()
 {
   for i in {1..10}; do
-    wget --tries 4 --retry-connrefused --waitretry=15 -qO- https://get.docker.com | sh
+    add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"

```

Comment:
I think you need to install the gpg key.
```sh
apt-get install -y apt-transport-https ca-certificates curl 
curl --max-time 60 -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - 
```

Reply:
Yes you're right thank you!